Firmware/Marlin_main.cpp

```diff
@@ -1440,13 +1440,6 @@ void setup()
     enable_z();
 #endif
 
-	farm_mode = eeprom_read_byte((uint8_t*)EEPROM_FARM_MODE);
-	if (farm_mode == 0xFF) farm_mode = false; //if farm_mode has not been stored to eeprom yet and farm number is set to zero or EEPROM is fresh, deactivate farm mode
-	if (farm_mode)
-	{
-		prusa_statistics(8);
```

Comment:
Removing the call to `prusa_statistics(8);` from this spot may result in problems with remote monitoring of the printer at the farm. The farm monitoring SW may accidentally expect 2 messages of "printer started" to consider a printer working.

`prusa_statistics(8);` sends "printer started", so it may be better to keep the call at this spot.

Reply:
@DRracer I will add this back. Is the comment below OK? A comment explaining the second call to `prusa_statistics(8)` is better than no explanation :)

```c
if (farm_mode) {
    // The farm monitoring SW may accidentally expect 
    // 2 messages of "printer started" to consider a printer working.
    prusa_statistics(8);
}
```